core/evancore.vue.libs/src/components/permissions/permissions.ts

```diff
@@ -150,12 +142,20 @@ class Permissions extends mixins(EvanComponent) {
   /**
    * Return translation for a certain key and scope if set, otherwise only the key.
    *
-   * @param key
+   * @param      {string}  keys    takes multiple keys and try to translation them against to the
+   *                               i18nScope. Last passed string will be used as default.
+   * @return     {string}  translation
    */
-  getTranslation(key: string): string {
-    const translated = this.$t(`${this.i18nScope}.${key}`);
+  getTranslation(...keys: string[]): string {
```

Comment:
what's the destructuring for?

Reply:
Yes correct, you can just put as many input parameters as you want. Ts supports it.